csrankings-j.csv

```diff
@@ -967,6 +967,7 @@ Jiajing Zhu,UESTC,https://sise.uestc.edu.cn/info/1036/11717.htm,NOSCHOLARPAGE
 Jiajun Bu,Zhejiang University,http://mypage.zju.edu.cn/bjj,OgZP2okAAAAJ
 Jiajun Chen 0001,Nanjing University,https://cs.nju.edu.cn/chenjiajun/index_en.htm,WIF7VaoAAAAJ
 Jiajun Liu,Renmin University of China,https://sites.google.com/site/drjiajunliu/homepage,xZGKQkcAAAAJ
+Jiajun Wen 0001,Shenzhen University,https://csse.szu.edu.cn/en/pages/user/index?id=835,VmiSYTgAAAAJ

```

Comment:
Senior Research Scientist, not eligible for inclusion.

Reply:
Hi, the current official position of Jiajun Wen is associated professor of College of Computer Science and Software Engineering
 in Shenzhen University, which is showed in the screenshot that marked in red box. This screenshot is from the official website page of College of Computer Science and Software Engineering in Shenzhen University. "Senior research Scientist" is just an honored title, not his official position. 
![screenshot ](https://github.com/user-attachments/assets/4553976e-6dde-42f2-8da6-ea7558f320e4)

![screenshot ](https://github.com/user-attachments/assets/b6ea5848-57c2-4923-9e02-f72c68eb4499)
[https://csse.szu.edu.cn/en/pages/user/index?id=835](url)

Reply:
Sorry I made mistakes when typed the position of Jiajun Wen in the previous comment. Jiajun Wen should be assistant professor, not associated professor like showing in the screenshot above. Please have a check. Thank you!